lib/webtrit_adapter_web/controllers/api/v1/user/history_controller.ex

```diff
@@ -96,6 +96,6 @@ defmodule WebtritAdapterWeb.Api.V1.User.HistoryController do
   end
 
   defp to_naive!(datetime, time_zone) do
-    datetime |> DateTime.shift_zone!(time_zone) |> DateTime.to_naive() |> NaiveDateTime.to_string()
+    datetime |> DateTime.shift_zone!(time_zone) |> DateTime.truncate(:second) |> DateTime.to_naive() |> NaiveDateTime.to_string()
```

Comment:
Could we add a comment above this line explaining the use of `truncate(:second)`? It could mention something along the lines of, 'PortaBilling cannot parse dates with millisecond precision' to provide clarity.